### PR TITLE
OWNERS: add TiDB cloud o11y team to owners

### LINKED ISSUE #209: Add TiDB cloud o11y team to owners file
Add TiDB cloud o11y team to owners file.

### PR DESCRIPTION
**Pull request by mornyx**
### What problem does this PR solve?

Issue Number: close #209 

### What is changed and how it works?

Add the following github accounts to owners:

- zhangpeijin-milo
- z2665
- just1900

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -25,3 +28,6 @@ reviewers:
   - ucjmh
   - zhangyangyu
   - zhongzc
+  - zhangpeijin-milo
+  - z2665
+  - just1900
```
Comment (wuhuizuo) [inline OWNERS:33]: Those lines are no need.
Approvers are also reviewers, they can give LGTMs for pull request.

Comment (mornyx) [reply OWNERS]: resolved

**Pull request merged**